src/components/molecules/ContentSearchbar.stories.js

```diff
@@ -0,0 +1,34 @@
+import { storiesOf } from "@storybook/vue";
+import { text } from "@storybook/addon-knobs";
+import { select } from "@storybook/addon-knobs";
```

Comment:
```suggestion
import { text, select } from "@storybook/addon-knobs";
```
you can combine these imports